RetakesAllocator/Menus/MenuManager.cs

```diff
@@ -12,7 +12,7 @@ public enum MenuType
     NextRoundVote,
 }
 
-public class MenuManager
+public class Menu_Manager
```

Comment:
Why is this renamed to `Menu_Manager`, follow the existing project naming convention.

Reply:
because of css has same name

https://github.com/roflmuffin/CounterStrikeSharp/blob/71c694b52ea32fe60a0972b9c775abb09fb52e96/managed/CounterStrikeSharp.API/Modules/Menu/MenuManager.cs#L21

Reply:
otherwise you will stuck with Obsolete("Use MenuManager.OpenChatMenu instead") cuz you have 2 same names

https://github.com/roflmuffin/CounterStrikeSharp/blob/71c694b52ea32fe60a0972b9c775abb09fb52e96/managed/CounterStrikeSharp.API/Modules/Menu/ChatMenu.cs#L83